storage/src/main/kotlin/maru/database/kv/KvDatabase.kt

```diff
@@ -59,7 +65,10 @@ class KvDatabase(
     }
   }
 
-  override fun getLatestBeaconState(): BeaconState = kvStoreAccessor.get(Schema.LatestBeaconState).get()
+  override fun getLatestBeaconState(): BeaconState =
+    kvStoreAccessor
+      .get(Schema.LatestBeaconState)
+      .getOrDefault(config.genesisState)
```

Comment:
This is not going to work with parent header validations, we need to actually put some initial state into DB